imap/jmap_calendar.c

```diff
@@ -86,8 +86,13 @@ static int jmap_calendarevent_query(struct jmap_req *req);
 static int jmap_calendarevent_set(struct jmap_req *req);
 static int jmap_calendarevent_copy(struct jmap_req *req);
 
-static int jmap_calendarevent_getblob(jmap_req_t *req, const char *blobid,
-                                      const char *accept);
+static int jmap_calendarevent_getblob(jmap_req_t *req,
```

Comment:
Nit: probably now is a good time to bundle some of these arguments in a getblob_req or the like?